server/api/assessments.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+const usersDb = require("../services/database/assessments");
```

Comment:
I suppose the `usersDb` is a remainder and should be named `assessmentsDb`, right?

Reply:
Done